### PR TITLE
fix: default img alt to "", accept isPageHeading, update tests

### PR DESCRIPTION
**Pull request by christian-dinh**
## Overview

applying posthumous feedback from https://github.com/Codecademy/client-modules/pull/1161, along with some other small tweaks
- default `imgAlt` to `""`
- accept `isPageHeading` prop and apply it to `LandingPageTitle`
- use markdown in story instead of html
- format tsdoc comments more goodly
- switch to RTL for tests, make them less bad

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: REACH-377
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: packages/gamut-labs/src/brand/LandingPageSection/LandingPageHero.tsx [hunk 1]
```diff
@@ -68,7 +73,9 @@ export const LandingPageHero: React.FC<LandingPageHeroProps> = ({
       }}
     >
       {title && (
-        <LandingPageSectionTitle isPageHeading>{title}</LandingPageSectionTitle>
+        <LandingPageSectionTitle isPageHeading={isPageHeading ?? false}>
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/brand/LandingPageSection/LandingPageHero.tsx:76]: This is part of why we often use `?: boolean` for falsy-accepting prop types, so you can remove the `?? false` in situations like that.

File: packages/gamut-labs/src/brand/LandingPageSection/LandingPageHero.tsx [hunk 2]
```diff
@@ -58,6 +62,7 @@ export const LandingPageHero: React.FC<LandingPageHeroProps> = ({
   ctaHref,
   imgSrc,
   imgAlt,
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/brand/LandingPageSection/LandingPageHero.tsx:64]: This is a little cleaner & easier to read IMO than a separate `defaultProps`:

```suggestion
  imgAlt = '',
```

**Review by JoshuaKGoldberg — COMMENTED** [review]
Most of the other feedback is from other people so I'll defer to them :)

File: packages/gamut-labs/src/brand/LandingPageSection/LandingPageHero.tsx (hunk 1 above)
Comment (christian-dinh) [reply packages/gamut-labs/src/brand/LandingPageSection/LandingPageHero.tsx:76]: This was actually to get around the fact that `isPageHeading` is required in `LandingPageSectionTitle`, but I can drop that constraint

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@5.6.1-alpha.9dc731.0<br/>@codecademy/styleguide@18.9.1-alpha.9dc731.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fb8373dc2bc95127681c152--gamut-preview.netlify.app>https://5fb8373dc2bc95127681c152--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fb8373dc2bc95127681c152>Deploy Logs</a></p>

**Review by rogervera — APPROVED** [review]
Looks good. Love the RTL refactor

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@5.7.0-alpha.eb506e.0<br/>@codecademy/styleguide@18.10.0-alpha.eb506e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fb8407d8ed60c23d04eec71--gamut-preview.netlify.app>https://5fb8407d8ed60c23d04eec71--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fb8407d8ed60c23d04eec71>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@5.7.1-alpha.2fe615.0<br/>@codecademy/styleguide@18.10.1-alpha.2fe615.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@5.7.1-alpha.e3cc93.0<br/>@codecademy/styleguide@18.10.1-alpha.e3cc93.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fb8421dc2bc95201381bdd1--gamut-preview.netlify.app>https://5fb8421dc2bc95201381bdd1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fb8421dc2bc95201381bdd1>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fb842ac9314af1e73a31b97--gamut-preview.netlify.app>https://5fb842ac9314af1e73a31b97--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fb842ac9314af1e73a31b97>Deploy Logs</a></p>

**Pull request merged**